src/wazuh_modules/wm_vulnerability_scanner.c

```diff
@@ -57,7 +57,16 @@ void* wm_vulnerability_scanner_main(wm_vulnerability_scanner_t * data)
 
         if (vulnerability_scanner_start_ptr)
         {
-            cJSON * config_json = wm_vulnerability_scanner_dump(data);
+            cJSON *config_json = cJSON_CreateObject();
+            cJSON_AddItemToObject(config_json, "vulnerability-detection", cJSON_Duplicate(data->vulnerability_detection, TRUE));
```

Comment:
Couldn't we do this? 
`cJSON *config_json = wm_vulnerability_scanner_dump(data);`

Reply:
@matias-braida  ping pring

Reply:
I did this way to keep building the cJSON config_json object explicitly